scripts/privyr-generic-cf-integration.js

```diff
@@ -0,0 +1,72 @@
+class PrivyrGenericCfIntegration {
+    constructor(license_code, form_id_or_name) {
+        this.license_code = license_code;
+        let self = this;
+        document.onreadystatechange = () => {
+            if (document.readyState === "complete") {
+                self.cform = document.getElementById(form_id_or_name) || document.getElementsByName(form_id_or_name);
+                window['_privyr_wpcf7'].captureLeads();
+
+                // removing random generated id on window close
+                // TODO: decide when to delete this id
+                window.onbeforeunload = () => {
+                    localStorage.removeItem('luid')
+                }
+            }
+        }
+    }
+
+    _uuidv4() {
+        return 'xxxxxxxx-xxxx-4xxx-yxxx-xxxxxxxxxxxx'.replace(/[xy]/g, function (c) {
+            var r = Math.random() * 16 | 0, v = c == 'x' ? r : (r & 0x3 | 0x8);
+            return v.toString(16);
+        });
+    }
+
+    _fetch_lead_user_id() {
+        let luid = localStorage.getItem('luid');
```

Comment:
can use sessionStorage instead of localStorage. benefits include - data is removed when when tab is closed

Reply:
DOne